BlockSettleUILib/Trading/RFQTicketXBT.cpp

```diff
@@ -658,6 +659,42 @@ bool RFQTicketXBT::checkBalance(double qty) const
    }
 }
 
+bool RFQTicketXBT::checkAuthAddr(double qty) const
+{
+   if (!ui_->authenticationAddressComboBox->isVisible()) {
+      return true;
+   }
+   else if (ui_->authenticationAddressComboBox->count() == 0) {
+      return false;
+   }
+
+   if (authAddressManager_->GetState(authAddr_)
+      == AddressVerificationState::Verified) {
+      return true;
+   }
+
+   const auto& tradeSettings = authAddressManager_->tradeSettings();
+   if (!tradeSettings) {
+      return false;
+   }
+
+   if (currentGroupType_ == ProductGroupType::XBTGroupType){
+      if (getProductToSpend() == UiUtils::XbtCurrency) {
+         return tradeSettings->xbtTier1Limit > bs::XBTAmount(qty).GetValue();
+      }
+      else {
+         QString indicativePrice = ui_->labelIndicativePrice->text();
+         QRegExp space(QLatin1String("\\s"));
+         indicativePrice.remove(space);
```

Comment:
I have 2 questions here:
* Why indicative price can contain spaces?
* Why can't you use `QString::remove(QChar)` instead of regexp?

Reply:
This is label which is contains number format string(e.g. 8 000.0, 12 000, 34, etc) and this is really depends on number style on system. Also the space character depend on configuration and not always equal to space character(code - 32), but also could be other like 160, etc. This is not save though, and temporary solution, I will create a task to proper propagate double indicative price to this widget and also to RFQDealer reply as well...

Reply:
As a quick and simple solution, you can just set/get double data for `ui_->labelIndicativePrice` field.

Reply:
Price string is created in a well-defined way. With accordance to locale and system settings
UiUtils::displayPriceForAssetType. And the space symbol is explicitly specified there.
static const QChar defaultGroupSeparatorChar = QLatin1Char(0xa0);